.travis.yml

```diff
@@ -1,14 +1,11 @@
 # This file was generated automatically from conda-smithy. To update this configuration,
 # update the conda-forge.yaml and/or the recipe/meta.yaml.
 
-language: objective-c
+language: generic
+
+os: osx
 
 env:
-  matrix:
-    
-    - CONDA_PY=27
-    - CONDA_PY=34
-    - CONDA_PY=35
```

Comment:
This actually might be ok. `python` normally needs to be a `build` and `run` dependency to get this.